cypress/component/Forms/forms.spec.jsx

```diff
@@ -675,6 +675,33 @@ describe('FormField - Tests', () => {
 
     });
 
+    it('default value can allow second entry of the same type', () => {
```

Comment:
What is the purpose of this test? To validate a correct behavior, or prove that the code is working incorrectly?

If it's to show a bug, I don't think it should be committed as a test. Instead I would add it to the bug ticket so we know how to reproduce it in the future.

Reply:
The purpose was to first figure out why I was seeing the behavior - if it was code I had in flight, or something core to the element.  Turns out that it's something core to the element.  I've left it in to demonstrate to other developers in the code base something that behaves unexpectedly.

I'll gladly move it to the ticket.